MAUI/PDF-Viewer/Migration.md

```diff
@@ -316,6 +316,39 @@ To make migration from [Xamarin SfPdfViewer](https://www.syncfusion.com/xamarin-
 <td>{{'[AnnotationSelected](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.PdfViewer.SfPdfViewer.html#Syncfusion_Maui_PdfViewer_SfPdfViewer_AnnotationSelected)'| markdownify }}</td>
 <td>Occurs when an annotation is selected.</td>
 </tr>
+<tr>
+<td>
+<div>{{'[PageChanged](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfPdfViewer.XForms.SfPdfViewer.html#Syncfusion_SfPdfViewer_XForms_SfPdfViewer_PageChanged)'| markdownify }},</div>
+
+</td>
+<td>{{'[PropertyChanged](https://learn.microsoft.com/en-us/dotnet/api/microsoft.maui.controls.bindableobject.propertychanged)'| markdownify }}</td>
+<td> Occurs when a property in the PDF Viewer is modified. It allows you to monitor page changes effectively. The example below demonstrates how to implement this:
```

Comment:
Does the C# tab works inside <td> block? Did you refer any other components?

Please check and correct this asap?
https://github.com/syncfusion-content/maui-docs/blob/ffcbced728a27a5fbc0c6a744dae37ee96f783a2/MAUI/PDF-Viewer/Migration.md

Reply:
I have referred in other control and removed the tab